modules/tinymce/src/core/main/ts/api/dom/DomQuery.ts

```diff
@@ -98,10 +98,10 @@ interface DomQuery<T extends Node = Node> extends Iterable<T> {
   attr (attrs: Record<string, string | boolean | number | null>): this;
   attr (name: string): string;
   before (content: DomQuerySelector<T>): this;
-  children (selector?: string): this;
+  children (selector?: string): DomQuery<ChildNode>;
```

Comment:
These were changed as they return a new `DomQuery` instance, not the current instance.